ext/ftp/ftp.stub.php

```diff
@@ -57,48 +57,48 @@ function ftp_systype($ftp): string|false {}
 
 /**
  * @param resource $ftp
- * @param resource $fp
+ * @param resource $stream
  */
-function ftp_fget($ftp, $fp, string $remote_file, int $mode = FTP_BINARY, int $resumepos = 0): bool {}
+function ftp_fget($ftp, $stream, string $filename, int $mode = FTP_BINARY, int $offset = 0): bool {}
```

Comment:
Only uncertainty I have left if we shouldn't be using `$remote_filename` here, due to the direct symmetry between ftp_fget and ftp_get (and similar for ftp_fput and ftp_put).

Reply:
Yes, this change would also address my initial problem.